queries.yaml

```diff
@@ -251,3 +251,9 @@ post_process:
       items_matching:
         kind: station
       rank_key: tile_rank
+  - fn: TileStache.Goodies.VecTiles.transform.remove_duplicate_features
+    params:
+      source_layers: [pois, buildings, landuse_labels]
```

Comment:
Because we duplicate landuse labels into both the landuse and landuse_labels layers, then this filter would end up removing one or the other layer's labels entirely. If you want to continue having labels in both landuse and landuse_layers, then we should copy them as a later step.

I forget why we are duplicating this information anyway?